env0/resource_template.go

```diff
@@ -208,6 +211,20 @@ func getTemplateSchema(prefix string) map[string]*schema.Schema {
 			ConflictsWith: allVCSAttributesBut("is_azure_devops", "token_id"),
 			RequiredWith:  requiredWith("token_id"),
 		},
+		"helm_chart_name": {
+			Type:          schema.TypeString,
+			Optional:      true,
+			Description:   "the helm chart name. Required if is_helm_repository is set to 'true'",
+			ConflictsWith: allVCSAttributesBut("helm_chart_name", "is_helm_repository"),
+		},
+		"is_helm_repository": {
```

Comment:
the `path` property isn't used when `is_helm_repository` is true.

could we block setting it?